src/components/Avatar/Styles/group.scss

```diff
@@ -18,3 +18,24 @@
     color: var(--text-secondary-color);
   }
 }
+
+.avatar-group-spaced {
```

Comment:
To reduce duplicate SASS, this may be moved into the above as

```
.avatar-group {
  ...
  &-spaced {...}
}
```

Then override only what's needed.